Zend/tests/assign_dim_obj_null_return.phpt

```diff
@@ -79,5 +79,5 @@ Cannot add element to the array as the next element is already occupied
 Illegal offset type
 Illegal offset type
 Cannot use a scalar value as an array
-Attempt to assign property 'foo' on bool
-Attempt to assign property 'foo' on bool
+Attempt to assign property "foo" on bool
```

Comment:
I've just realized that for bad array containers, we use `on value of type %s` instead of `on %s`. I'm not sure which format is preferred, but personally I find the former more accurate.